vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -3,54 +3,68 @@ sidebarDepth: 2
 ---
 
 # Create an Angular Micro Frontend
+This tutorial describes the process of building an Angular widget on Entando. It creates a docker-based bundle using the ent bundle CLI tool, which is the standard method starting with Entando 7.1.
 
-::: warning Prerequisites
-- [A working instance of Entando.](../../../docs/getting-started/README.md)
-:::
+Git-based bundles are also supported and to follow this tutorial in that format, see this [Angular tutorial](../../../../v7.0/tutorials/create/mfe/angular.md). 
 
-::: warning Tested Versions
-node v13.8.0 → We suggest using [nvm](https://github.com/nvm-sh/nvm) to handle node installations.
-:::
+## Prerequisites
+* [A working instance of Entando.](../../../docs/getting-started/README.md)
 
-## Create Angular App
+* node v13.8.0: Use [nvm](https://github.com/nvm-sh/nvm) to handle node installations.
 
-Install Angular CLI.
+## Initialize your Project Bundle
+1. To initialize your project bundle, give it any name you prefer and build the scaffolding
+```sh
+ent bundle init YOUR-BUNDLE
+``` 
+2. Migrate into the project bundle directory
+```sh
+cd YOUR-BUNDLE
+```
 
-``` bash
-npm install -g @angular/cli
+3. Add the first MFE with the name `angular-widget`
+```sh
+ent bundle mfe add angular-widget
 ```
 
-Generate a new angular application.
+## Create the Angular MFE
 
-``` bash
-ng new angular-widget
+1. Navigate into the `microfrontend` directory 
```

Comment:
error here, should be microfrontends

Reply:
end instructions followed by code with : for all steps